### DISCUSSION TITLE
How to Enforce Shorthand Properties?

### DISCUSSION THREAD

**Discussion by nxxkap** [chakra-ui/panda#2957] [Q&A] [answered]
Hello everyone,

I’m looking for advice on whether there’s a way to enforce shorthand properties within Panda CSS. Specifically, I’d like to know if it’s possible to restrict the use of standard CSS properties like background and instead enforce the use of shorthand properties like bg.

For example:

Restrict background but enforce bg instead.
Is there a recommended approach to implement this restriction within Panda CSS, or are there any built-in configurations or linter rules that could help enforce shorthand-only usage?

Thanks in advance for your guidance!

**Comment by EkeMinusYou** [comment, marked as answer]
How about trying eslint-plugin-panda [prefer-shorthand-properties](https://github.com/chakra-ui/eslint-plugin-panda/blob/main/docs/rules/prefer-shorthand-properties.md)?

**Comment by nxxkap** [comment]
@EkeMinusYou Thank you very much! It is exactly what I was looking for.
